pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -190,37 +188,37 @@ func makeWorkingDirScript(workingDirs map[string]bool) string {
 	return script
 }
 
-func makeWorkingDirInitializer(steps []corev1.Container) *corev1.Container {
-	workingDirs := make(map[string]bool)
+func makeWorkingDirInitializer(steps []v1alpha1.Step) *v1alpha1.Step {
+	workingDirs := map[string]struct{}{}
```

Comment:
why is `struct{}{}` better than `bool` here?

Reply:
Same as above, memory space